samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,202 @@
+
+**Microsoft Teams Meeting Helper sample app:** [Manifest](/samples/bot-ai-meeting-helper/demo-manifest/bot-ai-meeting-helper.zip)
+
+- To run the demo, you must set up and grant the necessary policy. Please follow the second step mentioned above for instructions on how to do this. 
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Table Storage](https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+- [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+- [PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-macos) if on macOS (`brew install --cask powershell`)
+- [Self-Signed Certificate](https://learn.microsoft.com/en-us/azure/key-vault/certificates/quick-create-portal)
+
+### Create an Azure Open AI service
+- In Azure portal, create a [Azure Open AI service](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
+- **Deploy Azure Open AI model:** Deploy the `gpt-35-turbo` model in your created Azure Open AI service for the application to perform translation.
+- Create and collect `AZURE_OPENAI_ENDPOINT`, `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_DEPLOYMENT_NAME`, and save those value  to update in `.env` file later.
+
+### Create an Azure Table Storage
+- In Azure portal, create a [Azure Table Storage](https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+- Create and collect `Account_Name`, `Account_Key`, `Table_Name`, and save those value  to update in `.env` file later.
+
+## Create an Azure Self-Signed Certificate
+- In Azure portal, create a [Self-Signed Certificate](https://learn.microsoft.com/en-us/azure/key-vault/certificates/quick-create-portal).
+- Create and collect `EncryptionCertificateId`, `Base64EncodedCertificate`, `PRIVATE_KEY_PATH`, and save those value  to update in `.env` file later.
+
+## Setup the Application Insights
+In Azure portal, create an [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/nodejs#resource) and save `Instrumentation Key` and `Connection String` values and which will be required later while updating `.env` file configuration in sample code.
+
+**Note>** While creating or setting up Application Insights, choose `NodeJS` as the application type.
+
+## Setup and use the sample locally 
+1) Navigate to the `samples/bot-ai-meeting-helper/.localConfigs` and update the values below.
```

Comment:
Here you have mentioned to update .localConfig but above steps are talking about .env files

Reply:
fixed